frontend/index.js

```diff
@@ -1,5 +1,17 @@
+const todaysDate = new Date()
+
 const dateHeader = document.getElementById('dateHeading');
+const dateToday = document.createTextNode(String(todaysDate.toLocaleString()));
+dateHeader.append(dateToday);
+
+const usersHoursFromGMT = (todaysDate.getTimezoneOffset())/60;
+const usersHoursFromKennedy = usersHoursFromGMT + 5;
+
+const kennedyTimeHeader = document.getElementById('kennedyTime');
+const kennedyTimeZoneOffset = document.createTextNode(usersHoursFromKennedy);
+kennedyTime.append(kennedyTimeZoneOffset);
 
-const dateToday = document.createTextNode(String(new Date()));
+const hoursAhead = document.createTextNode(' hours ahead of the Kennedy Space Centre');
+const hoursBehind = document.createTextNode(' hours behind the Kennedy Space Centre');
 
-dateHeader.append(dateToday);
\ No newline at end of file
+usersHoursFromKennedy > 0 ? kennedyTimeHeader.append(hoursAhead) : kennedyTimeHeader.append(hoursBehind);
```

Comment:
So for this file, I think the names are pretty much all fine! No problems there.

The thing I think could make a difference to readability is just to separate out the "today's date" and the "kennedy space center offset" bits of the code into two nicely named functions. This can involve a _tiny_ bit of repetition, but probably is worth it.

Lines 3 & 5 you could potentially remove the need for a variable by chaining the `append` directly on to the `getElementById` (and moving some lines of code around to compensate)

Lines 14-17 could potentially be condensed by doing a bit of string interpolation so you only work out the one string you wish to append?

Reply:
I agree. All changed. Thank you.